### DISCUSSION TITLE
Bumpkin Templates

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1407] [Art]
The purpose of this discussion is to provide artists with the dimensions/examples of the Bumpkin characters.

There have already been loads of creative submissions throughout Github and Discord and we are hoping to use this discussion to help keep record of all the submissions. When we launch Bumpkins we will need a range of community submissions and content to help the project.

There are no specific requests. We are encouraging all types of submissions from shirts, pants, hats, necklaces, wallpapers (backgrounds) and much more.

**Requirements**

- Dimensions of file should be 100x100px
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align
- A preview of the layer on top of a Bumpkin
- If submitting multiple items, an aseprite file would be appreciated to speed up review 🤗 

**Submissions**

There are no specific deadlines around submissions. We hope to keep this open for as long as needed to involve as many community artists as possible. Judging is TBD but we hope to use community consensus around submitted items. Chosen submissions will be rewarded with their designed item, accreditation and other contributing rewards ❤️ 

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service_

**Neutrality**

Any body part or item can be used with any style of Bumpkin.

It is important when designing your items that you keep them gender neutral and the colours look nice on the base layer Bumpkins.

If you are designing clothing, make sure you **test it on the green, beige and brown** base layers to ensure the colours look nice 🏆 

**Templates**

In this discussion you will find categories for each body part, including some examples.

You can use the attached `.ase` files to get a template of a farmer and goblin with all of the body parts included. This can help speed up the design process.

![farmer](https://user-images.githubusercontent.com/11745561/189238238-4e58cd91-c955-459c-87e6-f9d1d52603df.png)
![goblin](https://user-images.githubusercontent.com/11745561/189238241-ef821e46-da4c-4e45-8aca-ec01adb5b25f.png)

[aseprite.zip](https://github.com/sunflower-land/sunflower-land/files/9531018/aseprite.zip)

_All art provided is for use in Sunflower Land game and community only. Please be on the look out for scam projects and notify the team if you find anything 🙏 _

**Comment by adamhannigan** [comment]
Body templates.

Shirts, Pants, Necklaces, Hair etc. must align on any type of body. It is important when designing hair and hats to consider how it will look placed on top of a Goblin as well.

![Body_brown_2-100px](https://user-images.githubusercontent.com/11745561/189238350-aa0a7cee-5cf1-4b79-9f8c-f6dddec45c84.png)
![Body_green-100px](https://user-images.githubusercontent.com/11745561/189238355-9151f8fd-9b2a-4ced-a938-a3f97a1849ae.png)
![Body_beige-100px](https://user-images.githubusercontent.com/11745561/189238356-c61139c8-912c-4503-ab2d-13da6be8dd01.png)

**Reply by esp123** [reply]
![bumptemp](https://user-images.githubusercontent.com/74337946/189109441-62a626bc-a290-4ef7-95e7-06edecc6d9ed.png)
![gobtemp](https://user-images.githubusercontent.com/74337946/189109444-9e958801-2c55-433c-ac18-ee153c4f4503.png)

i made one from this

**Reply by JcEii** [reply]
### Left Hand Open

**Minified:**
![body1](https://user-images.githubusercontent.com/105348036/189249169-36a5b492-23f5-48cf-b98a-36622ea1e2d0.png)
![body2](https://user-images.githubusercontent.com/105348036/189249172-b46a70e5-cf43-4669-a1c6-9f88651d31f6.png)
![body3](https://user-images.githubusercontent.com/105348036/189249173-c9654f98-fe21-43a4-96a1-5b2a79c03eeb.png)
![body4](https://user-images.githubusercontent.com/105348036/189249376-89aeb48e-ad8c-4eef-97a9-f7cfb8068b6a.png)



**Aseprites:**
[body.zip](https://github.com/sunflower-land/sunflower-land/files/9531416/body.zip)

Farm no. 15747

**Reply by JcEii** [reply]
### Pointed Cheeks

**Minified:**
![body1](https://user-images.githubusercontent.com/105348036/189253506-361c0d64-26a3-4d47-8082-f4683763b117.png)
![body2](https://user-images.githubusercontent.com/105348036/189253512-7bb123b2-a3c6-4826-b1f1-c67debd91976.png)
![body3](https://user-images.githubusercontent.com/105348036/189253515-b5bd8603-d036-4c98-8d39-d3e9a561be42.png)
![body4](https://user-images.githubusercontent.com/105348036/189253517-cf9a303f-2f21-4cec-8249-feba59671ab5.png)

**Aseprites:**
[body2.zip](https://github.com/sunflower-land/sunflower-land/files/9531627/body2.zip)

Farm no. 15747

**Comment by adamhannigan** [comment]
Wallpapers.

A wallpaper will be an item that can be placed behind a Bumpkin.

![placeholder](https://user-images.githubusercontent.com/11745561/189061679-b748a845-8b47-4a43-81e9-8ae2bfa89ea6.png)

**Reply by JcEii** [reply]
### BG First Submission

**Preview:**
![bg](https://user-images.githubusercontent.com/105348036/194045558-36e22b39-ce5b-4f28-8919-64b3697828fc.gif)

**Name:**
1-Night island
2-Beach
3-Night seas
4-SFL Headquarters
5-Earth

_note: changed as of October 5, 2022_

Aseprites:
OLD - [bg 108x108.zip](https://github.com/sunflower-land/sunflower-land/files/9531265/bg.108x108.zip)
LATEST - [Redesigned_bg.zip](https://github.com/sunflower-land/sunflower-land/files/9715029/Redesigned_bg.zip)


Farm no. 15747

**Reply by esp123** [reply]
![tropic jungle](https://user-images.githubusercontent.com/74337946/189510034-f59b6ce1-9178-4d70-a7ca-0ae0a80a3cb8.png)
![sunset](https://user-images.githubusercontent.com/74337946/189510040-1198f0b5-4c74-4e7f-a4a1-69ce51d15be3.png)

Made a few bg.. please do criticize my work and tell me if im on the right track... Open for suggestion 😉

Discord: deeefault#0033

**Reply by shinooon** [reply]
Night Scene BG
![NIGHT SKY](https://user-images.githubusercontent.com/109294090/189906800-b5de8952-e68c-4dd4-9269-fccd3209e3ea.png)
![NIGHT SKY](https://user-images.githubusercontent.com/109294090/189904140-ae4570df-232f-4086-8949-dcd5eaa43d8e.png)

![NIGHT CLOUDS](https://user-images.githubusercontent.com/109294090/189906839-21a68bb8-7570-4f23-a6c2-00fab1c4d226.png)
![NIGHT CLOUD](https://user-images.githubusercontent.com/109294090/189904241-76fcdc01-047b-4cb6-9d0b-711b81742b4b.png)

![FOREST BG](https://user-images.githubusercontent.com/109294090/190545313-bffa7926-6dde-48a7-bb23-e1a20763088d.png)
![FOREST BG](https://user-images.githubusercontent.com/109294090/190545335-82fa9830-ad27-4749-b75d-85c4f24eec8c.png)



ps. it's my first time so feel free to criticize my work 🙂
farm #51738
discord: fam#4931

**Reply by adamhannigan** [reply]
@shinooon @JcEii @esp123 These are all amazing! Way beyond what I was expecting ❤️ 

We will definitely include some of these. Awesome work! 🤗

**Reply by FarmerKuenster** [reply]
Space planet background

![space-planet-bumpkin-background](https://user-images.githubusercontent.com/104322749/192422129-139afc78-b1e0-4d1b-89a4-ff6baa0cf2c8.png)
![400%-space-planet-bumpkin-background-export](https://user-images.githubusercontent.com/104322749/192422486-1b061cc6-2257-445c-8986-408f4415d6f6.png)

farrm# 32391
discord: farmerkunester#1626

**Comment by adamhannigan** [comment]
Shirts. When designing a shirt, ensure that it fills the torso and there is no gap between the shirt and the pants.

Example of a typical farmer shirt, and an apron style shirt.

![Chef_apron_01 - 100px](https://user-images.githubusercontent.com/11745561/189238494-4aca3639-eeee-4544-9a4c-4e6414ff526d.png)
![Shirt_red_white-100px](https://user-images.githubusercontent.com/11745561/189238497-69c822ed-ea53-4764-9034-74f28f53ac26.png)

Can place designs on top of this character: 
![farmer](https://user-images.githubusercontent.com/11745561/189238597-c7cedbd7-ad76-4a21-8e3a-59189c7c1618.png)

**Reply by JcEii** [reply]
### Old-fashion Coats

**Preview:**
![shirts](https://user-images.githubusercontent.com/105348036/189282045-208299d7-d434-4376-b53d-d064b218689e.gif)

Aseprites:
[shirts.zip](https://github.com/sunflower-land/sunflower-land/files/9532737/shirts.zip)

Farm no. 15747

**Reply by destinoextinto** [reply]
Medieval Landlord Goblin

![landlord_goblin_400](https://user-images.githubusercontent.com/83099564/189465540-d27b7580-bedc-443a-a49f-61a85c5b71a9.jpg)




Clothes + Hat + Face + Background in one file in different layers 

[landlord_goblin.zip](https://github.com/sunflower-land/sunflower-land/files/9539416/landlord_goblin.zip)

Farm #151314

**Reply by JcEii** [reply]
### Normal Design Shirts

**Preview:**
![normal design tshirt](https://user-images.githubusercontent.com/105348036/190655623-bd1d81c9-3426-4a06-8334-6ebd7e881934.gif)

1 - Chicken Tshirt
2 - Goblin Tshirt
3 - I love fishing Tshirt
4 - Pink fish Tshirt


Aseprites:
[normal design tshirts.zip](https://github.com/sunflower-land/sunflower-land/files/9584612/normal.design.tshirts.zip)

Farm no. 15747

**Comment by adamhannigan** [comment]
Pants. It is important when designing pants that there is no gap between the shirt and the pants.

We have 2 styles of pants: jeans and overalls. The overalls should overlay the shirt that a Bumpkin is wearing.


![Pant_overall_brown-100px](https://user-images.githubusercontent.com/11745561/189238711-d1dd5059-69cb-4542-a933-dbfab80684c3.png)

![Pant_jeans-100px](https://user-images.githubusercontent.com/11745561/189238706-d40d47e2-957f-453c-9840-b306b00f724c.png)

Example character to place pants on:

![farmer](https://user-images.githubusercontent.com/11745561/189238775-2a3a4572-1fb7-4f20-8233-9d25b6a9f44a.png)

**Reply by JcEii** [reply]
### Old-fashion pants

**Preview:**
![pants](https://user-images.githubusercontent.com/105348036/189244647-dcd6ae74-aabd-4fa4-ba16-c63eef17a288.gif)

Aseprites:
[pants.zip](https://github.com/sunflower-land/sunflower-land/files/9531284/pants.zip)

Farm no. 15747

**Comment by adamhannigan** [comment]
Shoes: A Bumpkin cannot farm without shoes!

![Shoes_black-100px](https://user-images.githubusercontent.com/11745561/189238808-4605f4bf-0395-4f66-98aa-52aab6678b73.png)
![Shoes_yellow-100px](https://user-images.githubusercontent.com/11745561/189238810-f2d59417-feac-491e-a749-ee88b7bdd0d2.png)

Example Bumpkin to design shoes on:

![farmer](https://user-images.githubusercontent.com/11745561/189238878-896dd0e1-7ef6-4fb7-8127-e861e705fad9.png)

**Reply by JcEii** [reply]
### Old-fashion boots

**Preview:**
![shoes](https://user-images.githubusercontent.com/105348036/189245023-02592691-28d4-4ec5-a8b1-6e61505346b5.gif)

**Aseprites:**
[shoes.zip](https://github.com/sunflower-land/sunflower-land/files/9531303/shoes.zip)

Farm no. 15747

**Reply by lelduck** [reply]
Frog boots
Disclaimer! No frogs where harmed in the making of this boots

![frogbootspreviewbig](https://user-images.githubusercontent.com/81706056/189853470-28c72133-fb9d-497d-89af-3ffd3f4403aa.png)
![frogbootspreview](https://user-images.githubusercontent.com/81706056/189853473-dbff0c3a-edc6-4829-a26b-4cb53c736cee.png)
![frogbootsbig](https://user-images.githubusercontent.com/81706056/189853477-2d46be99-7653-4918-aa8e-1957865a921c.png)
![frogboots](https://user-images.githubusercontent.com/81706056/189853478-1dbec903-b03f-4aef-abe4-818f5b6e3984.png)

**Reply by adamhannigan** [reply]
Love these frog boots! We actually tested them during a live stream. They look super cute ❤️  @lelduck 

You may want to consider adding a shadow to the bottom of the boots - when the green background is behind the farmer it is a bit difficult to see the boots

**Reply by stevew00dy** [reply]
OMG I didn't know I needed this in my life and now I can't live without them

**Reply by On1el** [reply]
![Кеды](https://user-images.githubusercontent.com/104461938/190706764-352dd95f-64c3-44cb-9619-45c699f1159a.png)
![Кедых10](https://user-images.githubusercontent.com/104461938/190706766-323e9da2-dd1b-480c-b570-0a972add3087.png)
![Пчелки](https://user-images.githubusercontent.com/104461938/190706789-a5915bb2-660d-4f5c-befd-2b365c4d24be.png)

**Comment by adamhannigan** [comment]
Eyes: Give your Bumpkin a sparkle!

![Eyes_02-100px](https://user-images.githubusercontent.com/11745561/189238931-1d0e12f7-f690-4c0f-997f-e5506c4d2671.png)
![Eyes_03-100px](https://user-images.githubusercontent.com/11745561/189238934-7e408dbb-2ade-455f-8887-c80b56521b8e.png)
![Eyes_04-100px](https://user-images.githubusercontent.com/11745561/189238935-9da7075a-2753-4803-9e92-0b8de83564a2.png)

Example Bumpkin to place eyes on:

![farmer](https://user-images.githubusercontent.com/11745561/189238980-2f17bc34-bc7f-4668-b907-ce3c89da5b2e.png)

**Reply by JcEii** [reply]
### Eyes Submissions

**Preview:**
Dark eye 
![eyes](https://user-images.githubusercontent.com/105348036/189258306-5298915a-36d2-46e7-aad9-6c309bd0d89d.gif) 
blue eye
![eyes2](https://user-images.githubusercontent.com/105348036/189258308-772d81e9-773c-4a86-bbad-af8aa92b3c1d.gif) 
green eye
![eyes3](https://user-images.githubusercontent.com/105348036/189258314-b2506208-30c7-4c2e-b256-529a7bf26afa.gif) 
Dandelion eye
![eyes4](https://user-images.githubusercontent.com/105348036/189258316-56321cb3-a7df-472b-978d-b336ce7d6579.gif) 
Heterochromia
![eyes5](https://user-images.githubusercontent.com/105348036/189258319-37a16bf6-4bf1-47d2-b89a-8fe979f16f3f.gif)

**Aseprites:**
[eyes.zip](https://github.com/sunflower-land/sunflower-land/files/9531794/eyes.zip)

Farm no. 15747

**Comment by adamhannigan** [comment]
Hair: Styles for everyone!

![Hair_03-100px](https://user-images.githubusercontent.com/11745561/189239036-767f0f6c-018a-4f93-a616-7b9cce98cf69.png)
![Hair_goblin-100px](https://user-images.githubusercontent.com/11745561/189239037-2c8fdda0-70d0-4600-a3a1-3d636c28184d.png)
![Hair_01-100px](https://user-images.githubusercontent.com/11745561/189239035-5ec2d58a-aea6-422d-b5ad-448285a1aa31.png)

Example Hairless Bumpkin to place hair on.

![farmer](https://user-images.githubusercontent.com/11745561/189239119-ba2d9ea5-2f1a-4fd3-93d9-a87829133799.png)

**Reply by JcEii** [reply]
### Hair Submissions

**Preview:**
![hair2](https://user-images.githubusercontent.com/105348036/194045081-634676c7-c9cc-4738-9574-69530f635f0b.gif)

_note: changed as of October 5, 2022_

**Aseprites:**
V1 - [hair.zip](https://github.com/sunflower-land/sunflower-land/files/9532829/hair.zip)
V2 - [hair2.zip](https://github.com/sunflower-land/sunflower-land/files/9715013/hair2.zip)

 
Farm no. 15747

**Reply by FarmerKuenster** [reply]
![hairstyle1-bumpkin](https://user-images.githubusercontent.com/104322749/192599291-02585d01-ffc1-4198-ad2c-08c3b3c0184b.png)


[hairstyle1-bumpkin.aseprite.zip](https://github.com/sunflower-land/sunflower-land/files/9658484/hairstyle1-bumpkin.aseprite.zip)


farrm# 32391
discord: farmerkunester#1626

**Reply by FarmerKuenster** [reply]
![hairstyle2-bumpkin](https://user-images.githubusercontent.com/104322749/192668253-cde956a4-e290-4aab-bf37-1e587504dbea.png)


[hairstyle2-bumpkin.aseprite.zip](https://github.com/sunflower-land/sunflower-land/files/9660693/hairstyle2-bumpkin.aseprite.zip)

farrm# 32391
discord: farmerkunester#1626

**Comment by adamhannigan** [comment]
Hats: Sit on top of a Bumpkin's hair. If needed, outside hair can be cropped.

![Chef_hat_01-100px](https://user-images.githubusercontent.com/11745561/189239170-87b0a4c8-d074-44cb-828d-8c9a81e1822d.png)

Example Bumpkin to place hat on top of.

![farmer](https://user-images.githubusercontent.com/11745561/189239197-0457a61d-3d20-42cd-b070-29d4ed99c060.png)

**Reply by JcEii** [reply]
### Hat Submissions

**Preview:**
![hat](https://user-images.githubusercontent.com/105348036/189309038-8b911374-c596-4671-a9ee-008410dc3c73.gif)

**Names:**
1-Straw hat
2-Goblin hat
3-orange stripe bandana
4-blue stripe bandana
5-japanese bandana
6-Salakot
7-Chicken hat

**Aseprites:**
[hat.zip](https://github.com/sunflower-land/sunflower-land/files/9533838/hat.zip)

Farm no. 15747

**Reply by adamhannigan** [reply]
@JcEii Loving this Goblin hat! 🚀  I can't wait to wear the skin of my victims on my head 😏

**Comment by adamhannigan** [comment]
Necklaces: The ultimate Bumpkin flex! 💪 

Necklaces sit on top of a Bumpkin's shirt and pants.

![Necklace_SFL-100px](https://user-images.githubusercontent.com/11745561/189239266-ae011f55-1e67-4470-a8d7-295b8c8fa1e2.png)

Example Bumpkin to place Necklace over:

![farmer](https://user-images.githubusercontent.com/11745561/189239299-cb0a2fea-93da-4445-99e0-2a2341367a4e.png)

**Reply by JcEii** [reply]
### Pearl Necklace

**Preview:**
![pearl necklace](https://user-images.githubusercontent.com/105348036/189372581-fe8dc1fd-036c-47ed-9bd4-4f624dfcc510.gif)

**Aseprites:**
[pearl necklace.zip](https://github.com/sunflower-land/sunflower-land/files/9536211/pearl.necklace.zip)

Farm no. 15747

**Comment by adamhannigan** [comment]
Mouths: Give your Bumpkin some character! ❤️ 

![Mouth_01-100px](https://user-images.githubusercontent.com/11745561/189239383-82b5bc54-8143-45fd-a0d9-71e9bedc78ea.png)
![Mouth_beard-100px](https://user-images.githubusercontent.com/11745561/189239386-7930a308-cce2-4477-b430-ade95f29800a.png)
![Mouth_goblin-100px](https://user-images.githubusercontent.com/11745561/189239388-6eb9ba0c-c527-4806-83a7-fd448ee60738.png)

Example mouthless Bumpkin to place on top of.

![farmer](https://user-images.githubusercontent.com/11745561/189239455-fb7a04fc-4ef6-4f73-b966-e123612a81bd.png)

**Reply by JcEii** [reply]
### Mouth Submission
**Preview:**
![mouth](https://user-images.githubusercontent.com/105348036/189380033-584994c7-9e38-4be1-bf81-dc085573c5fd.gif)

**Aseprites:**
[mouth.zip](https://github.com/sunflower-land/sunflower-land/files/9536499/mouth.zip)

Farm no. 15747

**Comment by On1el** [comment]
I apologize for attaching the files separately, it takes too long to unzip (and then it takes too long to assemble the image) 
![Girl](https://user-images.githubusercontent.com/104461938/190230309-1d618a33-033f-4829-b026-8410e0bccfdc.gif)
[Painting Girl.zip](https://github.com/sunflower-land/sunflower-land/files/9568837/Painting.Girl.zip)

**Comment by shinooon** [comment]
A Themed Clothing for Bumpkin

Preview: Bumpkin Holding Shepherd's Crook
![bumpkin farmer](https://user-images.githubusercontent.com/109294090/190855620-c1539ce3-7226-4362-a7c9-55de8b2992da.png)
![goblin farmer](https://user-images.githubusercontent.com/109294090/190855629-10ac82b8-fd77-418a-8298-31cbedd11669.png)
![bumpkin hat](https://user-images.githubusercontent.com/109294090/190650662-0c3596ab-0aba-40bd-939d-aedc1d056f24.png)
![goblin hat](https://user-images.githubusercontent.com/109294090/190650683-7697fcf8-d5ea-4dae-8674-df6b39303560.png)
![long sleeves](https://user-images.githubusercontent.com/109294090/190650747-3086b7b2-4cae-40d1-bc6d-84b7dd423276.png)
![blue pants](https://user-images.githubusercontent.com/109294090/190650776-1a9f905d-7574-491f-86fb-aa8308c8f01d.png)

Preview: Bumpkin Shepherd Set
![human shepherd set](https://user-images.githubusercontent.com/109294090/190855676-7b86b3df-3a11-4041-9d1c-ce1034b396ac.png)
![bumpkin shepherd](https://user-images.githubusercontent.com/109294090/190855694-d51444b5-fc8e-4bda-88eb-cf6b821f6fed.png)
![goblin shepherd set2](https://user-images.githubusercontent.com/109294090/190855700-bb57c381-a747-41ce-a775-29571cb61f24.png)
![human shepherd's hat](https://user-images.githubusercontent.com/109294090/190855724-f12df480-e31a-461a-9b79-466e2b021a2e.png)
![human shepherd's hat set2](https://user-images.githubusercontent.com/109294090/190855729-13bda03b-15e9-4b90-9ee7-b8eb5a5db770.png)
![goblin shepherd's hat](https://user-images.githubusercontent.com/109294090/190855736-fcc35183-9ed7-479a-bb7c-73453bd49cfc.png)
![goblin shepherd's hat set2](https://user-images.githubusercontent.com/109294090/190855741-c1bf5d08-6150-4d61-9a42-840b10e3e2fe.png)
![shepherd's clothe](https://user-images.githubusercontent.com/109294090/190855749-f78f857a-2ac3-4b31-8071-cdaf1b970e50.png)
![shepherd's robe](https://user-images.githubusercontent.com/109294090/190855756-2d5aada4-a6d0-4e31-ac4e-152ae8b0fc1b.png)
![shepherd's robe set2](https://user-images.githubusercontent.com/109294090/190855764-e78d04e4-a2ba-4d90-b1c8-d776f28e4fd8.png)
![sandals](https://user-images.githubusercontent.com/109294090/190855777-7a225145-5bb9-4a62-943b-e47755313b59.png)
![shepherd's crook](https://user-images.githubusercontent.com/109294090/190855783-5b96ee4f-0b08-4a5e-af77-c138431a9044.png)


Made a little adjustment for goblin hat to fit its head

farm #51738
discord: fam#4931

**Comment by clemrider** [comment]
So no farmers without farming suits ! Here my inspiration from where I'm living, where we produce Champagne !

![Background](https://user-images.githubusercontent.com/62701449/193415211-6a7d2f00-1a2d-4a2a-84be-c7fc42985825.png)
![GIF1](https://user-images.githubusercontent.com/62701449/193414357-a4a397a8-78db-4ada-af03-a1d3376d2d15.png)
![GIF2](https://user-images.githubusercontent.com/62701449/193414365-a9f8cc29-6505-4ef1-9f76-332449421461.png)
![example](https://user-images.githubusercontent.com/62701449/193415217-39b3d1f9-707c-48eb-b5fe-c9bc1e198949.png)
![A1Bottes](https://user-images.githubusercontent.com/62701449/193414805-a74e7aa6-2cf9-4d22-a86d-99b898829edf.png)
![A1BottesNoires](https://user-images.githubusercontent.com/62701449/193414829-1124516c-5033-4910-a69b-b5bf617a269a.png)
![A1VraisBottes](https://user-images.githubusercontent.com/62701449/193414878-54f8b053-d879-4446-8082-c3056c47310d.png)
![A1Cheveux2](https://user-images.githubusercontent.com/62701449/193414837-aaee0498-0b53-41e2-8b8b-1686fca6542e.png)
![A1CasquettesGrises](https://user-images.githubusercontent.com/62701449/193414830-464efbe1-5026-4162-8341-630ad97e3517.png)
![A1CheveuxChatains](https://user-images.githubusercontent.com/62701449/193414838-66386602-386a-4d34-bf1f-f2f865af03e6.png)
![A1Face](https://user-images.githubusercontent.com/62701449/193414844-bfb054dc-2a9b-4fc2-b72a-adccac4ef467.png)
![A1Pelle](https://user-images.githubusercontent.com/62701449/193414847-b4050a60-8b19-4af9-89c7-7208853ef854.png)
![A1TenueFermeOuverte](https://user-images.githubusercontent.com/62701449/193414864-c45d0801-ac05-49ac-ad49-4c22b935aa2a.png)
![A1TenueFermeOuverteBleue](https://user-images.githubusercontent.com/62701449/193414867-0eccb0e7-45ef-460f-abd0-ded56bdbe0b1.png)
![A1Grappe](https://user-images.githubusercontent.com/62701449/193414922-525f6511-02a3-4b4f-8478-ac48d6debc3c.png)








Farm#21662

**Comment by Asterion1989** [comment]
How much do you love icecream?

**Ice Cream**
Sweet food made from frozen milk. Recover 1 energy.
![icecream_1_ball](https://user-images.githubusercontent.com/78221179/195478569-494d68bc-5845-457c-b37b-98167af0289d.png)
**2 Balls Ice Cream**
Sweet food made from frozen milk. Recover 2 energy.
![icecream_2_balls](https://user-images.githubusercontent.com/78221179/195478573-2d5619ec-bea3-4884-8cfd-ee3116f04b8c.png)
**3 Balls Ice Cream**
Sweet food made from frozen milk. Recover 3 energy.
![icecream_3_balls](https://user-images.githubusercontent.com/78221179/195478574-9687ce26-98ef-4ca2-9576-ddb32d07c127.png)
![icecream](https://user-images.githubusercontent.com/78221179/195479099-a21f6038-bbbf-4678-a991-4919b71b6354.gif) 
![Capturar](https://user-images.githubusercontent.com/78221179/195479845-a23255a9-bc80-4d5f-abb4-36fa42763bfb.JPG)
[icecream_tool.zip](https://github.com/sunflower-land/sunflower-land/files/9769807/icecream_tool.zip)


Farm#55626

**Comment by Asterion1989** [comment]
Pumpkin Hat Evil
![pumpkin_hat_evil](https://user-images.githubusercontent.com/78221179/195487135-6fce2fd2-8884-48d2-b60b-94e1269e5989.png)
Pumpkin Hat Happy
![pumpkin_hat_happy](https://user-images.githubusercontent.com/78221179/195487137-baab90de-4250-431d-bcf5-52ecc7fd5be3.png)
![hat_01](https://user-images.githubusercontent.com/78221179/195487148-3ce7a0d1-f6a7-4c3e-ae2b-54c2ac7b26f9.png)
![hat_02](https://user-images.githubusercontent.com/78221179/195487149-c9445b75-edd8-4dd0-a68c-2796dfca7479.png)
[pumpkin_hat.zip](https://github.com/sunflower-land/sunflower-land/files/9769985/pumpkin_hat.zip)

Farm#55626

**Comment by On1el** [comment]
BEER
![Beer](https://user-images.githubusercontent.com/104461938/201756019-b0c7529c-1690-40b7-900c-4ddecef6ac40.png)
![Beerx5](https://user-images.githubusercontent.com/104461938/201756187-2ad5a542-ed3b-4a43-9a54-2c1fbd533f3f.png)
